streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -131,6 +131,9 @@
     public static final String ROCKSDB_CONFIG_SETTER_CLASS_CONFIG = "rocksdb.config.setter";
     public static final String ROCKSDB_CONFIG_SETTER_CLASS_DOC = "A Rocks DB config setter class that implements the <code>RocksDBConfigSetter</code> interface";
 
+    /** <code>cache.max.bytes.buffering</code> */
+    public static final String CACHE_MAX_BYTES_BUFFERING_CONFIG = "cache.max.bytes.buffering";
+    public static final String CACHE_MAX_BYTES_BUFFERING_DOC = "Maximum number of memory bytes to be used for buffering across all threads";
```

Comment:
How about: `The total bytes of memory the streams client can use across all threads to cache state store entries before flushing them to persistent storage. When some of these caching enabled persistent state stores are used in aggregate operators such as <code> KGroupedStream.aggregate(..) <code>, the updated aggregate result will no longer be sent out to the downstream operators upon each arrival of the incoming data, but only when the cache gets flushed due to task state committing for cache evictions. By doing this intermediate data volume of the aggregate updates can be reduced while the latency of aggregation output can be increased, hence users should choose this value carefully to make the appropriate tradeoff.`